Formula/passenger.rb

```diff
@@ -4,7 +4,7 @@ class Passenger < Formula
   url "https://github.com/phusion/passenger/releases/download/release-6.0.10/passenger-6.0.10.tar.gz"
   sha256 "3eb2c52652c96a136cc69cf342e3a61a8dd0e92cdbe9310a7aae249ab1abe0d4"
   license "MIT"
-  revision 1
+  revision 2
```

Comment:
Split this into a separate commit, please.

Reply:
@carlocab Done 👍🏻